examples/asa/package.json

```diff
@@ -7,7 +7,7 @@
     "chai": "^4.2.0"
   },
   "dependencies": {
-    "algosdk": "^1.8.0"
+    "algosdk": "https://github.com/algorand/js-algorand-sdk.git#9a5824782511dff1f521c68fe54d562b3872d1ae"
```

Comment:
Please use the syntax described in this article: https://medium.com/@jonchurch/use-github-branch-as-dependency-in-package-json-5eb609c81f1a

Reply:
Okay, thanks for suggestion

Reply:
Updated to `user/repo#commit` (`algorand/js-algorand-sdk.git#9a5824782511dff1f521c68fe54d562b3872d1ae`)